docs/source/examples/LanguageModelGeneration.md

```diff
@@ -0,0 +1,112 @@
+# Language Model Generation
```

Comment:
This needs to be added in the index: https://github.com/funboarder13920/OpenNMT-py/blob/lm/docs/source/index.rst